### DISCUSSION TITLE
Can we have a technical explanation for G SMOTE.

### DISCUSSION THREAD

**Discussion by Brainydaps** [georgedouzas/imbalanced-learn-extra#34] [Q&A]
How does G SMOTE generate synthetic data? We know the limitations of SMOTE is causing linearity, how does G SMOTE addresses this issue? Thanks in advance

**Comment by georgedouzas** [comment]
Hi @Brainydaps,

Have you read the Geometric SMOTE paper? What do you mean "SMOTE is causing linearity"?
